types/prompts.ts

```diff
@@ -27,8 +45,10 @@ export type PromptConfig<T extends GenericPromptResponse> = {
   when?: PromptWhen;
   pageSize?: number;
   default?: PromptOperand | ((answers: T) => PromptOperand);
+  transformer?: (input: string) => string | undefined;
   validate?:
     | ((answer?: string) => PromptOperand | Promise<PromptOperand>)
+    | ((answer?: number) => PromptOperand | Promise<PromptOperand>)
```

Comment:
can probably just make `answer` `any` too since we're still typing the individual validation functions

Reply:
+1 to @camden11 suggestion about generalizing, but instead of `any` I would throw `unknown` in there as an option to try before `any`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1293/commits/64ec64e3b160db98474cf96fff85513b85b11095